tests/testconfig.py

```diff
@@ -2,7 +2,7 @@
 oses = ["ubuntu", "macos"]
 
 # Defaults are conservative.
-parallel = False
+parallel = True
```

Comment:
This looks like a risky change. I would put this in a separate PR

Reply:
My bad, I didn't want this to be part of this PR. I'll remove it.